nowcasting_dataset/dataset/example.py

```diff
@@ -30,6 +30,13 @@ class Example(TypedDict):
     sat_x_coords: Array  #: OSGB geo-spatial coordinates.
     sat_y_coords: Array
 
+    # Topographic data
+    # Elevation map of the area covered by the satellite data
+    # Shape: [batch_size,] width, height, 1
```

Comment:
why isnt the shape just '[batch_size,] width, height'?

Reply:
It is, missed updating this description, fixed now!